libvcx/src/agency_comm/payload.rs

```diff
@@ -1,11 +1,10 @@
 use serde_json::Value;
 
+use agency_comm::get_message::MessagePayload;
+use agency_comm::message_type::*;
```

Comment:
I would prefer to avoid using `::*` in imports

Reply:
agree